web-client/netlify.toml

```diff
@@ -3,3 +3,9 @@
   to = "/index.html"
   status = 200
 
+[[headers]]
+  for = "/*"
+  [[headers.values]]
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
```

Comment:
This header is actually deprecated and we should not use/depend on this.

Reply:
I thought it was good practice to add even deprecated headers because of security in depth. They're ignored by modern browsers, but they don't cause issues in new ones.

Reply:
I would argue that the `X-Frame-Options` header matches your approach of considering in-depth security (which is awesome ❤️ ), as it's standardized and is still accepted by most older browsers, even though the CSP header setting is [accepted](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors) since over 10(?) years. So this specific header is up to us to keep/remove. Since we plan to handle this setting later in the CSP it could cause confusion (+ it can cause issues with Tauri apps).

The `X-XSS-Protection` was never standardized and is not even guaranteed to be useful in any old browser, in contrary it can be dangerous:

> Warning: Even though this feature can protect users of older web browsers that don't yet support [CSP](https://developer.mozilla.org/en-US/docs/Glossary/CSP), in some cases, XSS protection can create XSS vulnerabilities in otherwise safe websites. See the section below for more information.

[source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection)